src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0106Tests.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Azure.ClientSdk.Analyzers.Tests 
+{
+    public class AZC0106Tests 
+    {
+        private readonly DiagnosticAnalyzerRunner _runner = new DiagnosticAnalyzerRunner(new AsyncAnalyzer());
+
+        private const string TaskExtensionsString = @"
```

Comment:
Share this code.

Reply:
Do you want me to put it into separate file?